src/locale/locales/ca/messages.po

```diff
@@ -2451,7 +2451,7 @@ msgstr "Més informació sobre què és públic a Bluesky."
 
 #: src/components/moderation/ContentHider.tsx:152
 msgid "Learn more."
-msgstr ""
+msgstr "Mñes informació."
```

Comment:
Comparing to previous strings, it looks like there's a typo and the first word should be `Més`?